<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: rgb(255, 255, 255);
}
#save {
	background-color: rgb(223, 237, 245);
    border-style: solid;
    border-width: 1px;
	border-radius: 7px;
    padding: 1px 7px 2px;
    text-rendering: auto;
    color: initial;
    text-align: start;
    margin: 0em;
    font: 400 11px system-ui;
}
#save:hover {
	background-color: rgb(149, 190, 214);
	border: 1px  solid rgb(255, 255, 255);
	cursor: pointer;
}
#save:focus {
	background-color: rgb(75, 180, 240);
	border: 1px  solid rgb(255, 255, 255);
	cursor: pointer;
}
QLineEdit{
	padding: 5px;
	border: 1px solid ;
	color: rgb(0, 0, 0);
   	border-radius: 7px;
}</string>
  </property>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>70</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; font-style:italic;&quot;&gt;Name preset&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="null_value">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; font-style:italic; text-decoration: underline; color:#ff0000;&quot;&gt;Name can not be empty!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
